python/paddle/fluid/incubate/fleet/parameter_server/pslib/node.py

```diff
@@ -13,6 +13,7 @@
 """Defination of Server and Worker."""
 
 from . import ps_pb2 as pslib
+from six.moves import reduce
```

Comment:
it seems not using this  six reduce

Reply:
use it surely, under PY3, this will cause UT failure.